notebooks/least_squares/procedural_least_squares_driver.ipynb

```diff
@@ -356,9 +356,9 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.6"
+   "version": "3.8.8"
```

Comment:
I'm pretty sure this change is just a result from running the notebook locally. Can you undo it? (I need to figure out a good way for us to have notebooks with version control without running into this problem.)